packages/expandable-card/src/ExpandableCard/ExpandableCard.tsx

```diff
@@ -143,13 +144,8 @@ const ExpandableCard = ({
   );
 
   return (
-    <Card
-      darkMode={darkMode}
-      className={cx(cardStyle(darkMode), className)}
-      id={id}
-      {...rest}
-    >
-      <LeafyGreenProvider darkMode={darkMode}>
+    <LeafyGreenProvider darkMode={darkMode}>
```

Comment:
Why do we wrap this component with the provider?

Reply:
So that any component passed as a child can use darkMode.

Reply:
Do we do this with every compound component?

Reply:
I think so - at the top of my head, I know we use it for tooltip and modal